docs/getting-started/application/index.rst

```diff
@@ -151,39 +237,73 @@ when the actor is an instance of a ``User``:
     :language: polar
     :lines: 6-7
 
+.. tabs::
+    .. group-tab:: Python
 
-.. code-block:: console
+        .. code-block:: console
+
+            $ curl localhost:5000/whoami
+            <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+            <title>403 Forbidden</title>
+            <h1>Forbidden</h1>
+            <p>Not Authorized!</p>
+
+            $ curl -H "user: alice@foo.com"  localhost:5000/whoami
+            You are alice@foo.com, the CEO at Foo Industries. (User ID: 1)
 
-    $ curl localhost:5000/whoami
-    <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
-    <title>403 Forbidden</title>
-    <h1>Forbidden</h1>
-    <p>Not Authorized!</p>
+    .. group-tab:: Java
 
-    $ curl -H "user: alice@foo.com"  localhost:5000/whoami
-    You are alice@foo.com, the CEO at Foo Industries. (User ID: 1)
+        .. code-block:: console
+
+            $ curl -i localhost:5000/whoami
+            HTTP/1.1 401
```

Comment:
This will become a 403 with the prev suggestion, so just a note that the return value becomes:
```
{"timestamp":"2020-08-24T16:23:53.895+00:00","status":403,"error":"Forbidden","message":"oso authorization: unauthorized","path":"/whoami"}
```